libs/SalesforceSDKCore/SalesforceSDKCore/Classes/IDP/SFSDKAuthResponseCommand.m

```diff
@@ -60,4 +60,20 @@ - (NSString *)command {
     return @"authresponse";
 }
 
+- (NSString *)keychainReference {
+    return [self paramForKey:kSFKeychainReferenceParam];
+}
+
+- (void)setKeychainReference:(NSString *)keychain {
+    [self setParamForKey:keychain key:kSFKeychainReferenceParam];
+}
+
+- (NSString *)userHint {
```

Comment:
Isn't the auth response command going from SP to IDP (in response to a IDP initiated login request)?
If that's true, why is the user hint there?

Reply:
I'm not sure about the naming history, but it's used to open the SP app  here https://github.com/forcedotcom/SalesforceMobileSDK-iOS/blob/664d49497bcd6d02282a6c6d4fc853640c8981ac/libs/SalesforceSDKCore/SalesforceSDKCore/Classes/IDP/SFSDKIDPAuthHelper.m#L91

I was also thinking about breaking them into more specific commands instead of one, I could try aligning the naming more to Android